Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -7,8 +7,11 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		2506406E2304A2DE0028A9AC /* Dcrlibwallet.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2506406D2304A2DE0028A9AC /* Dcrlibwallet.framework */; };
-		B164982429AEDD7AF8C8782C /* Pods_Decred_Wallet.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = A9714AB438E597D403032748 /* Pods_Decred_Wallet.framework */; };
+		256CCAF7234640CF001208AE /* Dcrlibwallet.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 2569B2032343A5860070D9A2 /* Dcrlibwallet.framework */; };
+		257521D8233CBAF800F6F6E7 /* Dcrlibwallet.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 257521D7233CBAF800F6F6E7 /* Dcrlibwallet.framework */; };
```

Comment:
I have no idea why this reference keeps getting modified on your PRs, and why there are 2 here. Please fix.

Reply:
resolved in 174e2f3